app/helpers/Fetch.js

```diff
@@ -1,15 +1,6 @@
 import axios from 'axios';
 
 const Config = {
-  proxy: {
-    host: 'proxyserver.mepyd.gob.do',
-    port: 3128,
-    auth: {
-      username: 'covapp',
-      password: 'TIC..2k12*',
-    },
-  },
-  baseURL: 'https://proxyserver.mepyd.gob.do/',
   //url: '/',
   withCredentials: true,
   //httpAgent: new http.Agent({ keepAlive: true }),
```

Comment:
LGTM just remove these comments if they serve no purpose

Reply:
thanks Im on it